pkg/service/operation/storage/bolt.go

```diff
@@ -0,0 +1,38 @@
+package storage
+
+import (
+	"github.com/pkg/errors"
+	"github.com/tbd54566975/ssi-service/pkg/storage"
+)
+
+type BoltOperationStorage struct {
+	db *storage.BoltDB
+}
+
+func (b BoltOperationStorage) StoreOperation(op StoredOperation) error {
```

Comment:
is this PR intended to be a draft? or do you aim to merge with these unimplemented?

Reply:
I intend to merge with these unimplemented. They will be implemented in subsequent PRs. Just trying to keep the surface area small.